src/main/kotlin/lotto/domain/LottoesRank.kt

```diff
@@ -0,0 +1,15 @@
+package lotto.domain
+
+class LottoesRank(
+    private val value: Map<Rank, Int>
+) {
+    fun getRanks(): Map<Rank, Int> {
+        return value
+    }
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/kotlin-lotto/pull/104#discussion_r591543332

```suggestion
    val value: Map<Rank, Int>
) {
```